src/main/java/domain/chessGame/ChessBoard.java

```diff
@@ -0,0 +1,167 @@
+package domain.chessGame;
+
+import domain.piece.Color;
+import domain.piece.Pawn;
+import domain.piece.Piece;
+import domain.position.Path;
+import domain.position.Position;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public final class ChessBoard {
+
+    public static final int NUMBER_OF_KING_IN_UNFINISHED_GAME = 2;
+    private final Map<Position, Piece> chessBoard;
+    private Color turnOfColor;
+
+    public ChessBoard(Map<Position, Piece> chessBoard) {
+        this.chessBoard = new HashMap<>(chessBoard);
+        turnOfColor = Color.WHITE;
+    }
```

Comment:
```suggestion
    public ChessBoard(Map<Position, Piece> chessBoard) {
        this(chessBoard, Color.WHITE);
    }
```

요렇게 해서 중복을 줄일 수 있을 것 같네요 ㅎㅎㅎ